File: Beecrowd/1020 Idade em Dias/main.cpp
#include <iostream>

using namespace std;

int main() {

    int idade, ano, mes, dia;

    cin >> idade;

    ano = idade / 365;
    mes = (idade % 365) / 30;
    dia = (idade % 365) % 30;

    cout << ano << " ano(s)" << endl;
    cout << mes << " mes(es)" << endl;
    cout << dia << " dia(s)" << endl;

    return 0;
}

File: Beecrowd/1020 Idade em Dias/main.py
idade = int(input())

ano = idade / 365
mes = (idade % 365) / 30
dia = (idade % 365) % 30
	
print("{} ano(s)" .format(int(ano)))
print("{} mes(es)" .format(int(mes)))
print("{} dia(s)" .format(int(dia)))